src/main/java/com/ciandt/techgallery/service/enums/TechnologyOrderOptionEnum.java

```diff
@@ -0,0 +1,39 @@
+package com.ciandt.techgallery.service.enums;
+
+import com.ciandt.techgallery.utils.i18n.I18n;
+
+/**
+ * Enum for mapping Order Options.
+ * 
+ * @author Felipe Ibrahim
+ *
+ */
+public enum TechnologyOrderOptionEnum {
+
+  POSITIVE_RECOMENDATION_QUANTITY("Positive Recomendations Quantity"),
```

Comment:
This Enum is used to populate the combobox with order options in the front end. So I create in a separated Enum.